### PR TITLE
Alert frozen

### LINKED ISSUE #1212: Prompt on saving frozen sandbox
# 🌈 Feature

## What is this feature?
Prompt a user before forking a `frozen` sandbox.

## How the feature should work?
A user should be notified before forking a frozen sandbox.

## You have examples or an idea how it can be implemented?
Hovering over `?` button shows that saving frozen`

>  ...sandBox will fork on save

image

But the transition (when you click `ctrl + s`) is so subtle (and fast on a fast connection), it's hard to tell if you are working on a frozen SandBox or a forked on as you can see below.

We can prompt a user with a modal dialog to confirm if s/he wants to fork the frozen sandbox.

### PR DESCRIPTION
**Pull request by SaraVieira**
Alert when sandbox is frozen

Closes  #1212

### CODE REVIEW

File: packages/app/src/app/store/sequences.js [hunk 1]
```diff
@@ -172,15 +178,35 @@ export const forkSandbox = sequence('forkSandbox', [
...
+  },
+]);
+
 export const ensureOwnedEditable = sequence('ensureOwnedEditable', [
-  when(
-    state`editor.currentSandbox.owned`,
-    state`editor.currentSandbox.isFrozen`,
-    (owned, frozen) => !owned || frozen
-  ),
+  when(state`editor.currentSandbox.owned`, owned => !owned),
```
Comment (CompuIves) [inline packages/app/src/app/store/sequences.js]: Can we reverse this one? It's a bit confusing for me to see a negation here 😅

Comment (SaraVieira) [reply packages/app/src/app/store/sequences.js]: We can't reverse because of the old one checked if it was frozen too

What do you suggest?

Comment (CompuIves) [reply packages/app/src/app/store/sequences.js]: I'm thinking of this:

```
  when(state`editor.currentSandbox.owned`, owned => owned),
  {
  	true: [
      when(state`editor.currentSandbox.isFrozen`, isFrozen => isFrozen),
      {
        true: forkFrozenSandbox,
        false: [],
      },
    ],
    false: forkSandbox,
  },
```

Comment (CompuIves) [reply packages/app/src/app/store/sequences.js]: So flipping the true/false statements and setting `owned => owned`

Comment (SaraVieira) [reply packages/app/src/app/store/sequences.js]: Done :)

**Comment by CompuIves** [comment]
Great!

**Pull request merged**